### PR TITLE
scenario 3

### LINKED ISSUE #87: Finishing Scenario 3
- [ ] create the other versions (question 5 needs SEIRD, SEIRHD, and SIRHD models)
- [ ] Create a way to model detection rate (split I into I and I detected, the rate there is the detection rate)
- [ ] Finish question 4
- [ ] Do a hacky 3 way comparison by just plotting all 3

### PR DESCRIPTION
**Pull request by anandijain**
im going to commit frequently so stuff will need to get cleaned up but its probably better that this is visible

to close #87

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
> im going to commit frequently so stuff will need to get cleaned up but its probably better that this is visible

beautiful

File: Project.toml [hunk 1]
```diff
@@ -17,6 +18,7 @@ Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 SciMLBase = "0bca4576-84f4-4d90-8ffe-ffa030f20462"
 SciMLExpectations = "afe9f18d-7609-4d0e-b7b7-af0cb72b8ea8"
 Turing = "fce5fe82-541a-59a6-adf8-730c64b5f9a0"
+UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
```
[the remark this answers is not in the document]
Comment (anandijain) [reply Project.toml:21]: its a detail. ill remove it later once im done with the work

**Comment by anandijain** [comment]
to we have tools to nicely compare uncertainty forecasts?

**Comment by anandijain** [comment]
actually `Differential(t)(D) ~ (u_death / N) * H` is wrong. 

the original model (and my model) assume everyone dies in the hospital which doesnt seem right.

**Comment by ChrisRackauckas** [comment]
> to we have tools to nicely compare uncertainty forecasts?

Do we need more than https://sciml.github.io/EasyModelAnalysis.jl/dev/api/basic_queries/#Forecasting ?

**Comment by ChrisRackauckas** [comment]
> the original model (and my model) assume everyone dies in the hospital which doesnt seem right.

Ehh. we're supposed to just use the models as given 😅 we can get to that later.

**Comment by anandijain** [comment]
i think i just need to refactor the forecasts to return the `esol` and then make `get_uncertainty_forecast` do the `[esol[i][sym] for i in 1:samples]`, since i want to know what the pvals were that were sampled.

i don't understand this model. why is it that you can't recover in the hospital? am i reading the equations right?
```julia
 Differential(t)(S(t)) ~ -expo*I(t)*S(t)
 Differential(t)(E(t)) ~ expo*I(t)*S(t) - conv*E(t)
 Differential(t)(I(t)) ~ conv*E(t) - hosp*I(t) - rec*I(t)
 Differential(t)(R(t)) ~ rec*I(t)
 Differential(t)(H(t)) ~ hosp*I(t) - death*H(t)
 Differential(t)(D(t)) ~ death*H(t)
```

**Comment by ChrisRackauckas** [comment]
It's a rough disease haha

yeah we can fix that tomorrow. I have a code dump of the other two models in petri net form coming.

**Comment by anandijain** [comment]
i don't know how to properly do the detection model.

**Comment by anandijain** [comment]
okay to start, with:
* S->E->IU->ID->H->D
* IU->R
* ID->R 

we get that if detection rate is high we will see more people die, which doesn't make sense
so i could add `IU->D` but then i need to add a parameter or use `u_death` 

if we allow 
* IU->H (undetected to directly be hospitalized)

then it seems like detection rate wouldnt do anything. 

so maybe the right way is to add IU->D

**Comment by anandijain** [comment]
but that doesn't really make sense because then you would want ID->D too, 

maybe add IU->D and you say the detection rate and hosp rate are the same. so you get rid of ID as a state.
then all infected people are undetected infections. 

so then D(I) ~ e*E*I-h*I-d*I-r*I. but then its messed up that you cant recover in the hospital. 
okay so then ill just add H->R

qualitatively then, hospitalized are infected but cannot infect, but can recover.

sorry, i know im kinda fumbling the ball here

**Comment by anandijain** [comment]
okay i think 
```
Differential(t)(S(t)) ~ -(u_expo / N) * I(t) * S(t)
Differential(t)(E(t)) ~ (u_expo / N) * I(t) * S(t) - (u_conv / N) * E(t)
Differential(t)(I(t)) ~ (u_conv / N) * E(t) - (u_hosp / N) * I(t) - (u_rec / N) * I(t) - (u_death / N) * I(t)
Differential(t)(R(t)) ~ (u_rec / N) * I(t) + (u_rec / N)*H
Differential(t)(H(t)) ~ (u_hosp / N) * I(t) - (u_death / N) * H(t) - (u_rec / N)*H
Differential(t)(D(t)) ~ (u_death / N) * H(t) + (u_death / N) * I(t)
```
might be right i need to try it though

**Comment by anandijain** [comment]
its not quite ready but im marking ready. 
hopefully easymodelanalysis finishes precompiling in time for me to do a test run

**Pull request merged**